soccer/gameplay/evaluation/defensive_positioning.py

```diff
@@ -60,25 +63,29 @@ def get_points_from_rect(rect, step=0.5):
 #
 # @param ignore_robots: Ignore these robots in defensive calculations
 # @return Returns the best position to cover an error
-def create_area_defense_zones(ignore_robots=[]):
+def create_area_defense_zones(
+    ignore_robots: Optional[List[robocup.Robot]] = None
+) -> Optional[robocup.Point]:
+    if ignore_robots is None:
+        ignore_robots = []
```

Comment:
Was something complaining about an empty list?

Reply:
pycharm complains about `Default argument is mutable`, though I don't think there were any cases were this was used incorrectly. I can revert this back

Reply:
I prefer reverting just because doing this optional -> list conversion is a bunch of boiler plate that hides what it's really doing.

Reply:
ok done